Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/conflict.py

```diff
@@ -39,9 +42,40 @@ def parser(cls, parser, loggers=None):
         )
 
     @classmethod
-    def find_conflict_pr(cls, remote, branch):
-        for pr in remote.pull_requests.find(head=branch, opened=True):
-            return pr
+    def radar_for_id(cls, radar_id):
+        for tracker in Tracker._trackers:
+            if isinstance(tracker, radar.Tracker):
+                return tracker().client.radar_for_id(radar_id, additional_fields=['sourceChanges'])
+
+    @classmethod
+    def find_conflict_pr(cls, remote, radar_id):
+        """
+        Because we don't know what the target branch was just given the radar id,
+        we need to search for PRs with branches that start with the integration prefix
+        and have the first and last sha.
+        """
+        radar_obj = cls.radar_for_id(radar_id)
+        assert radar_obj, 'Could not fetch radar object for id {}'.format(radar_id)
+        shas = []
+
+        repo_name = remote.name if '/' not in remote.name else remote.name.split('/', 1)[-1]
+        for entry in radar_obj.sourceChanges.splitlines():
```

Comment:
I think `radar_obj.source_changes` can give you a list right off the bat, assuming you use `Tracker.from_string` to create an issue object

Reply:
This property doesn't seem valid, requesting `source_changes` vs `sourceChanges` from API returns an error, and I don't see this listed in the radar client docs.